kstreams-live-update/aggregator/src/main/java/io/debezium/examples/kstreams/liveupdate/aggregator/cdi/EagerExtension.java

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
I must say I am not happy with all the CDI extension stuff in the demo as it is more or less distraction from the main point of the example. Cannot we use `@Initialized` observer to achive the same functionality?

Reply:
I hear what you're saying, but it's the best I could come up with. If you got ideas for improving it, why not. Otherwise I think it's fine as is.